app/src/components/GHGIHomePage/Hero.tsx

```diff
@@ -112,7 +125,7 @@ export function Hero({
                         display="flex"
                       >
                         <span data-testid="hero-city-name">
-                          {inventory?.city?.name}
+                          {inventory ? inventory?.city?.name : city?.name}
```

Comment:
Could this logic be simplified by creating a variable definition about the return block?
E.g. `const activeCity = inventory?.city ?? city;`
This way you don't need to have all of the ternary operators everywhere here.